organization/urls.py

```diff
@@ -10,7 +10,11 @@
     url(regex=r'/<slug:slug>/update/',
         view=views.UpdateOrganization.as_view(),
         name='update_organization'),
+    url(regex=r'/list/',
+        view=views.ListAllUserOrganizations.as_view(),
```

Comment:
Can you rename this view to `ListOrganization` and not use the `/list/`, leave the url empty as we do in the other urls, like `deck/urls.py`.